app/actions/websocket/index.ts

```diff
@@ -172,13 +172,12 @@ export async function handleEvent(serverUrl: string, msg: WebSocketMessage) {
             handleLeaveTeamEvent(serverUrl, msg);
             break;
         case WebsocketEvents.UPDATE_TEAM:
+            handleUpdateTeamEvent(serverUrl, msg);
             break;
-
-        // return dispatch(handleUpdateTeamEvent(msg));
         case WebsocketEvents.ADDED_TO_TEAM:
+            handleTeamAddedEvent(serverUrl, msg);
```

Comment:
Nit: Maybe rename to something like "handleUserAddedToTeamEvent"?

Reply:
done.